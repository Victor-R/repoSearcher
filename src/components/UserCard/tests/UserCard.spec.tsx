import { render } from "@testing-library/react";
import { UserCard } from "..";

describe("Components: UserCard", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <UserCard
        active
        name="Gandalf"
        imageUrl="https://i.kym-cdn.com/entries/icons/original/000/042/690/Screen_Shot_2022-11-16_at_2.24.03_PM.jpg"
      />
    );

    expect(container).toMatchSnapshot();
  });
});
