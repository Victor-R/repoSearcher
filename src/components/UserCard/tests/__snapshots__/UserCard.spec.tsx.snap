// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Components: UserCard > should match snapshot 1`] = `
<div>
  <div
    class="sc-bczSft lgVgBH"
  >
    <img
      class="sc-gsnScm ginXVy"
      src="https://i.kym-cdn.com/entries/icons/original/000/042/690/Screen_Shot_2022-11-16_at_2.24.03_PM.jpg"
    />
    <h3
      class="sc-dkzCjb gHZaNP"
    >
      Gandalf
    </h3>
  </div>
</div>
`;
